.github/compatibility-test-matrices/release-v3.4.x/transfer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "chain-a": ["release-v3.4.x", "v3.3.1", "release-v2.5.x", "v2.4.2"],
```

Comment:
I am adding to tips of release branches here, so that we can test that transfer with memo works between `v3.4.x` and `v2.5.x`, but I am unsure if this is supported by the e2e framework, so let's give it a try and see what happens...

Reply:
ah I left my below comments before seeing this. This one might be tricky as the first part of the compatibility test process is building the the `release-x.y.x` image. In this case some of those images won't exist. As a work around you could launch those compatibility tests for the images that don't exist manually then cancel them after the image is built.